doc/howtos/pull-request-guidelines.rst

```diff
@@ -168,3 +168,7 @@ Dos and Don'ts
   happens only for the full set of commits that are getting merged, so
   when bisecting history, the merge commits are good candidates for
   testing because they are more likely to build correctly.
+
+* Never add backdoors (= unexpected access to the system), debug features,
```

Comment:
The proposed rewording (IMHO) incorrectly talks about "debug features for unexpected access" - "debug features" are never allowed in the default configuration, regardless whether they enabled unexpected access or not.

How about:

* Never add backdoors (i.e., something that enables unexpected and unauthorized access to the system
  or some of its features) or debug features to the default configuration. Such capabilities may be enabled
  only in explicit debug configurations of the OS images.